css/properties.json

```
@@ -9851,7 +9867,7 @@
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/visibility"
   },
   "white-space": {
-    "syntax": "normal | pre | nowrap | pre-wrap | pre-line | break-spaces",
+    "syntax": "normal | pre | nowrap | pre-wrap | pre-line | break-spaces | [ <'white-space-collapse'> || <'text-wrap'> ]",
```

From your notes:

>there is another related property, white-space-trim, which is specced as being included in the white-space shorthand value, but this is not supported yet so I have not added it. Let me know if you think I should add it.

My opinion is to add it here for completeness sake, and we're going to have to add it in future anyway. Protocol on that looks like something in BCD (a note?) for browser-specific details, what do you reckon?

Adding the suggestion for your convenience:

```suggestion
    "syntax": "normal | pre | nowrap | pre-wrap | pre-line | break-spaces | [ <'white-space-collapse'> || <'text-wrap'> || <'white-space-trim'> ]",
```

spec link: https://www.w3.org/TR/css-text-4/#white-space-property